app/assets/javascripts/cited_documents.js

```diff
@@ -8,6 +8,9 @@
   CitedDocuments = {
     init: function (el) {
       var $el = $(el);
+      const citedDocList = $el.find('.cited-documents-list');
+      if (citedDocList[0].innerHTML != '') return;
```

Comment:
Can you add a comment to the code about why we're bailing out here and in which situations this happens?

Reply:
Done